package.json

```diff
@@ -27,7 +33,7 @@
   },
   "scripts": {
     "dev": "parcel demo/index.html -d .devserver --open",
-    "build": "node rollup.build.js && bundlesize",
+    "build": "npx babel-node rollup.build.js && bundlesize",
```

Comment:
You don't need to transpile it with babel then.

Reply:
Whoops, fixed (including the `@babel/node` dep. I added).